agent/consul/intention_endpoint.go

```diff
@@ -145,7 +145,7 @@ func (s *Intention) prepareApplyUpdate(ident structs.ACLIdentity, authz acl.Auth
 	}
 
 	// make sure we set the hash prior to raft application
-	args.Intention.SetHash(true)
+	args.Intention.SetHash()
```

Comment:
Not that you need to but I bet we could remove the boolean from all the other SetHash functions.

Reply:
Oh nice! I'll look at doing that in a separate cleanup